src/client/containers/NextStepsPage/NextStepsPage.component.js

```diff
@@ -1,3 +1,140 @@
 import React from 'react';
+import PageHeader from '../../components/PageHeader/PageHeader.component';
+import SideMenu from '../../components/SideMenu/SideMenu.component';
+import Buttons from '../../components/Buttons/Buttons.component';
+import rediLogo from '../../assets/images/redi-logo.png';
+import hyfLogo from '../../assets/images/hyf-logo.png';
+import udacityImg from '../../assets/images/Next_page_imgs/udacity.png';
+import courseraImg from '../../assets/images/Next_page_imgs/coursera.png';
+import linkedinImg from '../../assets/images/Next_page_imgs/linkedinLearn.png';
+import lyndaImg from '../../assets/images/Next_page_imgs/lynda.png';
+import cfImg from '../../assets/images/Next_page_imgs/cf.png';
+import udemyImg from '../../assets/images/Next_page_imgs/udemy.png';
+import roundImg from '../../assets/images/Next_page_imgs/round.png';
+import { useHistory, NavLink } from 'react-router-dom';
+import './NextStepsPage.styles.css';
 
-export const NextStepsPage = () => <section>Next Steps Page</section>;
+export const NextStepsPage = () => {
+  const history = useHistory();
+
+  function handleClick() {
+    history.push('/elevator-pitch');
+  }
+  return (
+    <div className="main-part-next-page">
+      <div>
+        <PageHeader />
+      </div>
+      <div className="middle-part">
+        <div className="sidemenu-nextpage">
+          <SideMenu />
+        </div>
+        <div className="orange-box" />
+        <div className="white-box">
+          <h1>YOUR NEXT STEPS</h1>
+          <p>
+            <strong>Suggestions for Upgrading your Knowledge :</strong>
+          </p>
+          <p>Checkout our schools</p>
+          <div className="main-images">
+            <div className="redi-image-next-page">
+              <NavLink to="https://www.redi-school.org">
+                <img
+                  src={rediLogo}
+                  alt="Redischool"
```

Comment:
The alt value can have spaces so please write the name of the school which is ReDI School